internal/fourslash/fourslash.go

```diff
@@ -28,13 +29,47 @@ type FourslashTest struct {
 	in     *lspWriter
 	out    *lspReader
 	id     int32
+	vfs    vfs.FS
 
-	testData *TestData
+	testData *TestData // !!! consolidate test files from test data and script info
```

Comment:
Weirdly some requests in Strada's fourslash use the original files instead of the fourslash object's script info with the updated content, so I'm not sure how things are gonna go and have kept the whole test data here for now.